### PR TITLE
DAMD-71: TargetType should be defined in datamodel.txt and headers

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: python/pfs/instmodel/lightSource.py [hunk 1]
```diff
@@ -3,18 +3,17 @@
...
 from .spectrum import FlatSpectrum
-from .spectrum import SlopeSpectrum
 from .spectrum import ConstantSpectrum
 from .spectrum import NullSpectrum
 from .spectrum import PfsSimSpectrum
 from .spectrum import AmbreSpectrum
 
 
-__all__ = ["fluxForPhotons", "fluxDensityForPhotons", "DomeStatus", "LightSource"]
+__all__ = ["fluxForPhotons", "fluxDensityForPhotons", "LightSource"]
```
Comment (hassanxp) [inline python/pfs/instmodel/lightSource.py]: Why this change?

Comment (PaulPrice) [reply python/pfs/instmodel/lightSource.py]: It doesn't exist.
